cli/stream_command.go

```diff
@@ -2232,6 +2324,17 @@ func (c *streamCmd) prepareConfig(_ *fisk.ParseContext, requireSize bool) api.St
 		}
 	}
 
+	if (c.subjectTransformSource != "" && c.subjectTransformDest == "") || (c.subjectTransformSource == "" && c.subjectTransformDest != "") {
+		fisk.Fatalf("must specify both --transform-source and --transform-destination")
+	}
+
+	if c.subjectTransformSource != "" && c.subjectTransformDest != "" {
```

Comment:
this if is redundant then - you would always error out above if this case is false.

Reply:
The test is not there to check both are not "" (that's taken care of in the checkRepubTransform) but to check if they are set or not in which case they need to be included in the generated cfg.

I could check just on one since I know if one is set the other is as well, but that felt 'not right' so I did that check on both 🤷‍♂️

Reply:
ok yeah you're right, not working for 3 months rots the brain :)

Reply:
If that's the reason, then I'm jealous of your brain rot :)